srcpkgs/gcalcli/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'gcalcli'
+pkgname=gcalcli
+version=4.3.0
+revision=1
+build_style=python3-pep517
+hostmakedepends="python3-setuptools python3-wheel"
+depends="python3-dateutil python3-google-api-python-client
+	python3-httplib2 python3-oauth2client python3-parsedatetime
+	python3-vobject"
```

Comment:
```suggestion
 python3-httplib2 python3-oauth2client python3-parsedatetime
 python3-vobject"
```

line continuations should be indented with a single space, not a tab

Reply:
Fixed.